PublicDateAtUSN: 2017-04-04 14:00:00 UTC
Candidate: CVE-2017-7233
CRD: 2017-04-04 14:00:00 UTC
PublicDate: 2017-04-04 17:59:00 UTC
References: 
 https://www.djangoproject.com/weblog/2017/apr/04/security-releases/
 https://ubuntu.com/security/notices/USN-3254-1
 https://www.cve.org/CVERecord?id=CVE-2017-7233
Description:
 Django 1.10 before 1.10.7, 1.9 before 1.9.13, and 1.8 before 1.8.18 relies
 on user input in some cases to redirect the user to an "on success" URL.
 The security check for these redirects (namely
 ``django.utils.http.is_safe_url()``) considered some numeric URLs "safe"
 when they shouldn't be, aka an open redirect vulnerability. Also, if a
 developer relies on ``is_safe_url()`` to provide safe redirect targets and
 puts such a URL into a link, they could suffer from an XSS attack.
Ubuntu-Description: 
Notes: 
Bugs: 
Priority: medium
Discovered-by:
Assigned-to:
CVSS:
 nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N [6.1 MEDIUM]

Patches_python-django:
upstream_python-django: released (1.8.18)
precise_python-django: released (1.3.1-4ubuntu1.23)
trusty_python-django: released (1.6.11-0ubuntu1.1)
trusty/esm_python-django: released (1.6.11-0ubuntu1.1)
esm-infra-legacy/trusty_python-django: released (1.6.11-0ubuntu1.1)
vivid/stable-phone-overlay_python-django: DNE
vivid/ubuntu-core_python-django: DNE
xenial_python-django: released (1.8.7-1ubuntu5.5)
esm-infra/xenial_python-django: released (1.8.7-1ubuntu5.5)
yakkety_python-django: released (1.8.7-1ubuntu8.2)
devel_python-django: released (1.8.7-1ubuntu11)
